src/library/scala/collection/StringOps.scala

```diff
@@ -157,6 +157,21 @@ object StringOps {
   }
 }
 
+/** Provides extension methods for strings.
+  * 
+  * Some of these methods treat strings as a plain collection of [[Char]]s
+  * without any regard for Unicode handling. Unless the user takes Unicode
+  * handling in to account or makes sure the strings don't require such handling,
+  * these methods may result in unpaired or invalidly paired surrogate code
+  * units.
+  * 
+  * @define unicodeunaware This method treats a string as a plain sequence of
+  *                        Char code units and makes no attempt to keep
+  *                        surrogate pairs or codepoint sequences together.
+  *                        The user is responsible for making sure such cases
+  *                        are handled correctly, and may otherwise result in
+  *                        an invalid Unicode string.
```

Comment:
This is gramatically a bit strange, I read it as "The user ... may otherwise result in an invalid Unicode string".

Reply:
Indeed. Hopefully this is better.